### ISSUE TITLE
Technical discovery: Back to top UX in learning center

### ISSUE THREAD

**Issue by jenniferlee-dsva** [department-of-veterans-affairs/va.gov-team#13646]
## Technical discovery: Back to top UX in learning center 

This is for mvp 1.1 learning center templates. 

Please work with @rtwell on the desired FE design and UX  for implementation. 

**Basic requirement:** 
Each article page should enable a dynamic back to top link. 
This back to top should appear based on page height/length.

**Comment by jenniferlee-dsva** [comment]
fysa - @brianalloyd @MickinSahni  - I didn't add milestone for this because it can be pulled into a future sprint. Just wanted to make sure we don't forget to work on this functionality which will be completely FE. 

If we need to triage - it's a `nice to have` for LC 1.0; `required `for 1.1.

**Comment by rtwell** [comment]
if it helps: https://github.com/department-of-veterans-affairs/vets-design-system-documentation/issues/298

**Comment by ncksllvn** [comment]
The JS behavior on this will be really simple, so I'm not worried about that. If we have complex style to make this "back to top" button sticky on different areas of the screen, that could get a little complicated.

With the request to the design system still in early stages, should we consider the tech discovery and implementation effectively blocked?

**Comment by rtwell** [comment]
i created the #298 on the design system board for awareness for that group (and what will be that team). But i think this team will build the interaction and then the design system team can include it in the system. Or am I overstepping/misunderstanding who "owns" which part of building something like this?

**Comment by ncksllvn** [comment]
I think it's fine to work on this as-is. It just makes me nervous to have a design system conversation happening in parallel because of the potential of those processes overlapping with our product's processes.

**Comment by rtwell** [comment]
don't be skerd! the good news is i'm on both teams and guiding the work, so i can make sure there is no overlap

**Comment by jenniferlee-dsva** [comment]
Trying to add this OG ticket to Brian's new epics but zenhub not letting me rn: 
https://app.zenhub.com/workspaces/vft-59c95ae5fda7577a9b3184f8/issues/department-of-veterans-affairs/va.gov-team/14340
And
https://app.zenhub.com/workspaces/vft-59c95ae5fda7577a9b3184f8/issues/department-of-veterans-affairs/va.gov-team/14347

**Comment by brianalloyd** [comment]
@jenniferlee-dsva added to new Epic [#14347](https://app.zenhub.com/workspaces/vft-59c95ae5fda7577a9b3184f8/issues/department-of-veterans-affairs/va.gov-team/14347) and closed out #14340 as I combined with #14347 as discussed.  Also adding it to Sprint 32 as a stretch and can push to Sprint 33 if needed.

**Comment by kelsonic** [comment]
Just realized this is the UX ticket for [this dev implementation ticket](https://github.com/department-of-veterans-affairs/va.gov-team/issues/14347), sorry about renaming the title 🙂 So I believe the design directly conflicts with using ForeSee, is that correct? Not sure if this ticket needs iterating then to find a solution that doesn't conflict with getting user feedback per page:

![image](https://user-images.githubusercontent.com/12773166/97339969-d4b0ca80-1848-11eb-81df-5cd669701fae.png)

![image](https://user-images.githubusercontent.com/12773166/97339989-dc706f00-1848-11eb-8708-ccf7bc25bcf2.png)

**Comment by rtwell** [comment]
We (or maybe I should say "I")  imagined the Foresee element and BTT element would never coexist on the same page, and teams would strategize which pages would get which. If that is not an option here, then we need to rethink the BTT design/implementation (which I do have a backup!). @jenniferlee-dsva are we using Foresee in lieu of the Yes/No bits? Apologies for my out-of-the-loop-ness.

**Comment by jenniferlee-dsva** [comment]
We are not using foresee tab+modal widget. 

We (ryan, liz, and I) are meeting Monday to see how we can inorporate foresee backend with our current fe article rating component. We will wait for Medallia if for some reason this is not feasible.

**Comment by ncksllvn** [comment]
Since we've started implementation per https://github.com/department-of-veterans-affairs/va.gov-team/issues/14347, can we close this?

**Comment by jenniferlee-dsva** [comment]
Sounds all right to me.
